### PR TITLE
148 Consistent German in 'Meiner Bewerbungen' index and edit action from pupils view

### PR DESCRIPTION
**Pull request by radscheit**
(#148) Please have a look at my last comment in the US.

### CODE REVIEW

File: config/locales/de.default.yml [hunk 1]
```diff
@@ -106,41 +106,41 @@ de:
       second: Sekunde
       year: Jahr
   errors:
-    format: "%{attribute} %{message}"
+    format: '%{attribute} %{message}'
     form_invalid: 
-      one: "In einem Feld ist ein Fehler aufgetreten"
-      other: "In %{count} Feldern sind Fehler aufgetreten"
+      one: "Achtung. In einem Feld ist ein Fehler aufgetreten. Korrigiere die Fehler um fortzufahren."
```
Comment (Baschdl) [inline config/locales/de.default.yml]: "den Fehler" instead of "die Fehler"

Comment (radscheit) [inline config/locales/de.default.yml]: Check.

File: config/locales/de.default.yml [hunk 2]
```diff
@@ -106,41 +106,41 @@ de:
...
-      greater_than_or_equal_to: muss größer oder gleich %{count} sein
-      inclusion: ist kein gültiger Wert
-      invalid: ist nicht gültig
-      less_than: muss kleiner als %{count} sein
-      less_than_or_equal_to: muss kleiner oder gleich %{count} sein
+      accepted: muss akzeptiert werden.
+      blank: 'Feld ist leer. Bitte fülle es aus, um fortfahren zu können.'
+      present: darf nicht ausgefüllt werden.
+      confirmation: stimmt nicht mit %{attribute} überein.
+      empty: Feld muss ausgefüllt werden.
```
Comment (Blaidd-Drwg) [inline config/locales/de.default.yml]: Why the "Feld"? Inconsistent with other messages.
What is the difference between "empty" and "blank"? When is the message for "empty" shown)?

**Comment by Blaidd-Drwg** [comment]
screenshot from 2017-01-01 13-33-51
This still doesn't look right to a German reader. Maybe we can make the name of the field bold and leave out the "Feld"? As in:
**Telefonnummer für Notfälle** ist leer. Bitte fülle es aus, um fortfahren zu können.

File: config/locales/de.default.yml [hunk 3]
```diff
@@ -106,41 +106,41 @@ de:
...
     form_invalid: 
-      one: "Achtung. In einem Feld ist ein Fehler aufgetreten. Korrigiere den Fehler um fortzufahren."
-      other: "Achtung. In %{count} Feldern sind Fehler aufgetreten. Korrigiere die Fehler um fortzufahren."
+      one: "Achtung. In einem Feld ist ein Fehler aufgetreten. Korrigiere den Fehler, um fortzufahren"
+      other: "Achtung. In %{count} Feldern sind Fehler aufgetreten. Korrigiere die Fehler, um fortzufahren"
     messages:
-      accepted: muss akzeptiert werden.
-      blank: 'Feld ist leer. Bitte fülle es aus, um fortfahren zu können.'
-      present: darf nicht ausgefüllt werden.
+      accepted: muss akzeptiert werde
```
Comment (Blaidd-Drwg) [inline config/locales/de.default.yml]: Missing n

File: config/locales/de.default.yml [hunk 4]
```diff
@@ -106,41 +106,41 @@ de:
...
+      blank: 'Feld ist leer. Bitte fülle es aus, um fortzufahren'
+      present: darf nicht ausgefüllt werden
       confirmation: stimmt nicht mit %{attribute} überein.
```
Comment (Blaidd-Drwg) [inline config/locales/de.default.yml]: Still has a full stop

File: config/locales/de.default.yml [hunk 5]
```diff
@@ -106,41 +106,41 @@ de:
...
-      less_than_or_equal_to: muss kleiner oder gleich %{count} sein.
+      empty: 'Feld ist leer. Bitte fülle es aus, um fortzufahren'
+      equal_to: muss genau %{count} sein
+      even: muss gerade sein
+      exclusion: ist nicht verfügbar
+      greater_than: muss mindestens %{count} sein
+      greater_than_or_equal_to: darf höchstens %{count} sein
+      inclusion: ist kein gültiger Wert
+      invalid: ist nicht gültig
+      less_than: draf höchstens %{count} sein
```
Comment (Blaidd-Drwg) [inline config/locales/de.default.yml]: *darf

File: config/locales/de.default.yml [hunk 6]
```diff
@@ -106,41 +106,41 @@ de:
...
+      less_than_or_equal_to: muss mindestens %{count} sein
```
Comment (Blaidd-Drwg) [inline config/locales/de.default.yml]: ... why are the messages for <= and > identical? :sweat_smile: 
Same for >= and <.

File: config/locales/de.default.yml [hunk 7]
```diff
@@ -107,24 +107,25 @@ de:
...
+      blank: 'Feld ist leer: bitte fülle es aus, um fortzufahren'
       present: darf nicht ausgefüllt werden
       confirmation: stimmt nicht mit %{attribute} überein.
       empty: 'Feld ist leer. Bitte fülle es aus, um fortzufahren'
       equal_to: muss genau %{count} sein
       even: muss gerade sein
       exclusion: ist nicht verfügbar
-      greater_than: muss mindestens %{count} sein
-      greater_than_or_equal_to: darf höchstens %{count} sein
+      greater_than: muss grüßer als %{count} sein
```
Comment (Blaidd-Drwg) [inline config/locales/de.default.yml]: :wink:

Comment (radscheit) [inline config/locales/de.default.yml]: Oh der ist peinlich. 👎  Danke :)

File: config/locales/de.application_letters.yml [hunk 8]
```diff
@@ -16,7 +16,7 @@ de:
       Programmierkenntnisse sind keine  Voraussetzung zur Teilnahme. Diese Information
       wird zur Gruppeneinteilung benötigt."
     login_before_creation: "Du musst angemeldet sein, um dich für Workshops bewerben zu können."
-    successful_update: "Bewerbung erfolgreich bearbeitet"
+    successful_update: "Die Bewerbung wurde erfolgreich aktualisiert"
```
Comment (pinussilvestrus) [inline config/locales/de.application_letters.yml]: Maybe it seems inconsistent if there is an full sentence at this point and on "successful_deletion" and "successful_creation" there are just word groups

**Pull request merged**